src/pages/Select.vue

```diff
@@ -112,20 +110,15 @@ export default {
         };
     },
     created() {
+        fetch(this.endpoint + "/journey-parents")
+            .then((x) => x.json())
+            .then((x) => {
+                this.categories = x.map((c) => ({ ...c, selected: false }));
+            });
         fetch(this.endpoint + "/journeys")
             .then((x) => x.json())
             .then((x) => {
                 this.journeys = x.map((j) => ({ ...j, selected: false }));
-                let uniqueParents = Array.from(
-                    new Set(
-                        x.map((journey) => {
-                            return journey.parent;
-                        })
-                    )
-                );
-                this.categories = uniqueParents.map((parent) => {
-                    return { name: parent, selected: false };
-                });
                 this.loading = false;
```

Comment:
We seem to be setting loading to false, even if the categories aren't loaded?

Reply:
Sorry - for some reason I thought I'd made this a nested fetch. 
I've updated it to make both fetch calls using promise.all()